### PR TITLE
Add an Admin UI and API to test sending emails using AWS SES

### PR DESCRIPTION
**Pull request by kryswisnaskas**
## Description of change
Adds a section in the admin UI where sending emails using AWS SES can be tested. Since cloud.gov doesn't support adding  DKIM for its domain associated with non-production environments, we need to run a test in production. The goal is to deploy to production and test sending emails from the prod domain.

## How to test
- On Dev navigate to the Email section in the admin UI
- Fill in the form with a valid email address
- Note that there is an error
- Use the following address: 
success@simulator.amazonses.com
- Verify there is a success message or there is a successful entry in the log file
## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1924

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [n/a] Code is meaningfully tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by GarrettEHill** [comment]
Needs tests

**Comment by kryswisnaskas** [comment]
> Needs tests

The idea is to push this test to production as soon as possible so we can determine whether any other configuration or setup is needed. Once the test is successful and we decide to go with SES, we can remove the testing code.

File: src/routes/apiDirectory.js [hunk 1]
```diff
@@ -76,6 +77,7 @@ router.use('/goal-templates', goalTemplatesRouter);
 router.use('/events', eventRouter);
 router.use('/session-reports', sessionReportsRouter);
 router.use('/national-center', nationalCenterRouter);
+router.use('/email', emailRouter);
```
Comment (thewatermethod) [inline src/routes/apiDirectory.js:80]: Do we need to expose 2 routes? I think we are only using the one in ```/src/routes/admin/index.js``` and we could delete this one, which doesn't seem to check any permissions

**Comment by thewatermethod** [comment]
Test worked exactly as it was supposed to, I think we should kill the extra route before pushing to prod

File: frontend/src/pages/Admin/Email.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const [success, setSuccess] = useState();
+
+  const onSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const formData = new FormData(e.target);
+      const plainFormData = Object.fromEntries(formData.entries());
+
+      await sendEmail(plainFormData);
+      setSuccess('Email successfully sent');
```
[the remark this answers is not in the document]
Comment (kryswisnaskas) [reply frontend/src/pages/Admin/Email.js:22]: Added.

**Pull request merged**